.github/workflows/ubuntu.yml

```diff
@@ -131,14 +164,25 @@ jobs:
           [ -d "${LOCAL_BUILDS}/rnp-build/src/tests" ]
           [ ! -d "${LOCAL_BUILDS}/rnp-build/src/tests/googletest-build" ]
           [ ! -d "${LOCAL_BUILDS}/rnp-build/src/tests/googletest-src" ]
+          
   cmake-disable-rubyrnp:
-    env:
-      DOWNLOAD_RUBYRNP: Off
-      RNP_TESTS: rnp_tests.hash_test_success
-    runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, 'skip ci')"
-    timeout-minutes: 50
-    steps:
+   strategy:
+      matrix:
+        cc: ["gcc"]
```

Comment:
Same here - no need to recheck with different compilers, as we are just making sure that CMake define works correctly.

Reply:
Same as above - it really tests ci scripts

Reply:
These tests are commented